at_functional_test/runLocal.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+echo "Getting dependencies" && dart pub get
+
+cd test || exit
```

Comment:
Rather than cding around could we just use something like `docker compose -f test/docker-compose.yml up`?